docs/user_guide/use_case/custom_dataloaders.md

```diff
@@ -3,3 +3,113 @@
 :::{note}
 This page is under construction.
 :::
+
+In SCVI, custom dataloaders allow you to create a tailored data pipeline that can handle unique formats or complex datasets not covered by the default loaders. A custom dataloader can be useful when you have a specific structure for your data or need to preprocess it in a particular way before feeding it into the model, in order to gain some advantage.
+
+For example, we offer custom dataloaders that do not necessarily store the data on memory while training, thus enable us to expand the sizes of dataset that we can train our models based on while not being limited by the amount of memory that we have.
```

Comment:
Change to: While scvi-tools allows disk-backed training, custom dataloaders support a collection of AnnData stored on disk and also optimize performance for disk-backed loading.

Reply:
Sorry next sentence:
but change:
thus enable us to expand the sizes of dataset that we can train our models based on while not being limited by the amount of memory that we have -> which enables training on a group of AnnData datasets and allows training on datasets that are to large for the amount of memory.